File: php/scripts/get_travel_restrictions.php
<?php
require_once(__DIR__."/../vendor/autoload.php");
require_once("utils.php");
use Namelivia\TravelPerk\ServiceProvider;
use Carbon\Carbon;
$isSandbox = false;
$travelperk = (new ServiceProvider())->build(getenv("API_KEY"), false);
$restriction = $travelperk->travelSafe()->travelSafe()->travelRestrictions(
	"ES",
	"FR",
	"country_code",
	"country_code",
	Carbon::today()
);
write_output("travel_restrictions", [
	$restriction->origin->name,
	$restriction->origin->type,
	$restriction->origin->countryCode,
	$restriction->destination->name,
	$restriction->destination->type,
	$restriction->destination->countryCode,
	$restriction->authorizationStatus,
	$restriction->summary,
	$restriction->details,
	$restriction->startDate,
	$restriction->endDate,
	$restriction->updatedAt,
	$restriction->requirements,
	$restriction->requirements[0]->category->id,
	$restriction->requirements[0]->category->name,
	$restriction->requirements[0]->subCategory->id,
	$restriction->requirements[0]->subCategory->name,
	$restriction->requirements[0]->summary,
	$restriction->requirements[0]->details,
	$restriction->requirements[0]->startDate,
	$restriction->requirements[0]->endDate,
	$restriction->requirements[0]->documents,
]);

File: python/scripts/get_travel_restrictions.py
import os
import datetime
from travelperk_http_python.builder.builder import build
from utils import write_output
travelperk = build(os.environ["API_KEY"], False)
restriction = travelperk.travelsafe().travelsafe().travel_restrictions(
    "ES", "FR", "country_code", "country_code", datetime.date.today()
)
write_output("travel_restrictions", [
    restriction.origin.name,
    restriction.origin.type,
    restriction.origin.country_code,
    restriction.destination.name,
    restriction.destination.type,
    restriction.destination.country_code,
    restriction.authorization_status,
    restriction.summary,
    restriction.details,
    restriction.start_date,
    restriction.end_date,
    restriction.updated_at,
    restriction.requirements,
    restriction.requirements[0].category.id,
    restriction.requirements[0].category.name,
    restriction.requirements[0].sub_category.id,
    restriction.requirements[0].sub_category.name,
    restriction.requirements[0].summary,
    restriction.requirements[0].details,
    restriction.requirements[0].start_date,
    restriction.requirements[0].end_date,
    restriction.requirements[0].documents,
])
